docs/cugraph/source/conf.py

```diff
@@ -209,7 +208,9 @@ def setup(app):
 )
 
 breathe_projects = {
+    'libcugraph': '../libcugraph/_xml',
```

Comment:
```suggestion
    'libcugraph': os.environ['XML_DIR_LIBCUGRAPH'],
```

Looks like this was missed from my previous diff.